app/controllers/idv/in_person/address_controller.rb

```diff
@@ -60,6 +78,14 @@ def analytics_arguments
         }
       end
 
+      def redirect_to_next_page
+        if updating_address?
+          redirect_to idv_in_person_ssn_url
+        else
+          redirect_to idv_in_person_verify_info_url
+        end
```

Comment:
I would expect this to be reversed - if they are updating their address, they came from verify_info and need to return there.

Reply:
I agree. I think it is working because the ssn page is seeing if it has the data and then redirecting. I bet if you opened up network tab you would see an extra redirect.